### ISSUE TITLE
$or bug

### ISSUE THREAD

**Issue by sylvainlap** [feathersjs/feathers-knex#120]
Hi,

I'm using the last version of `feathers-knex`.

Here is my `hook.params.query`:
```
{
    $or: [
        { firstname: { $like: "dace" },
        { lastname: { $like: "dace" },
    ],
    company: 1
}
```

And here is how `feathers-knex` transforms it:
```
"select "users".* from "users" where "company" = 1 or ("firstname" like 'dace') or ("lastname" like 'dace') limit 10"
```

As you can see, the `or` after the `company = 1` should be a AND, not a OR.

**Comment by daffl** [comment]
This is probably happening in https://github.com/feathersjs/feathers-knex/blob/master/src/index.js#L90 but I'm not sure how it should be changed. Does order matter?

**Comment by sylvainlap** [comment]
According to knexjs docs, the `or` conditions should be all wrapped into a `.where()`.

For example:
```
knex('users').where(function() {
  this.where('id', 1).orWhere('id', '>', 10)
}).orWhere({name: 'Tester'})

Outputs:

select * from "users" where ("id" = 1 or "id" > 10) or ("name" = 'Tester')
```

**Comment by sylvainlap** [comment]
I created a PR for this issue: https://github.com/feathersjs/feathers-knex/pull/122

**Comment by daffl** [comment]
Closed via #122
